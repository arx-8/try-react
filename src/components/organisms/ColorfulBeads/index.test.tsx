import { _ColorfulBeads as SutComponent } from "."
import { create } from "react-test-renderer"
import React from "react"

describe("ColorfulBeads", () => {
  it("3 Beads", () => {
    // ## Arrange ##
    // ## Act ##
    const result = create(<SutComponent beadsCount={3} />)
    // ## Assert ##
    expect(result).toMatchSnapshot()
  })
})
