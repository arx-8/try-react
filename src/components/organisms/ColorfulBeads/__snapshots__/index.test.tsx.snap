// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorfulBeads 3 Beads 1`] = `
.emotion-0 {
  margin-top: 40px;
}

<div
  className="emotion-0"
>
  <div
    className="ui red circular label"
    onClick={[Function]}
  />
  <div
    className="ui orange circular label"
    onClick={[Function]}
  />
  <div
    className="ui yellow circular label"
    onClick={[Function]}
  />
</div>
`;
